scripts/create_test

```diff
@@ -319,13 +324,13 @@ def create_test(testargs, compiler, machine_name, no_run, no_build, no_batch, te
                     machine_name = testsplit[4]
                 else:
                     expect(machine_name == testsplit[4],
-                           "abiguity in machine, please use the --machine option")
+                           "ambiguity in machine, please use the --machine option")
             if testsplit[5] is not None:
                 if compiler is None:
                     compiler = testsplit[5]
                 else:
-                    expect(compiler == testsplit[5],
-                           "abiguity in compiler, please use the --compiler option")
+                    logger.info("Switch compiler %s to %s"%(compiler, testsplit[5]))
```

Comment:
I've changed it so that this feature is only used with the --xml options and so will not conflict with acme's style of compare and generate usage.